Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerPerformanceTrackerTests.swift

```diff
@@ -107,62 +107,84 @@ class SentryUIViewControllerPerformanceTrackerTests: XCTestCase {
             let spans = self.getStack(tracker)
             transactionSpan = spans.first
             
-            let blockSpan = spans.last!
-            XCTAssertEqual(blockSpan.context.parentSpanId, transactionSpan.context.spanId)
-            XCTAssertEqual(blockSpan.context.spanDescription, self.loadView)
+            if let blockSpan = spans.last, let transactionSpan = transactionSpan {
+                XCTAssertEqual(blockSpan.context.parentSpanId, transactionSpan.context.spanId)
+                XCTAssertEqual(blockSpan.context.spanDescription, self.loadView)
+            } else {
+                XCTFail("Expected spans")
+            }
             callbackExpectation.fulfill()
         }
-        XCTAssertEqual((transactionSpan as! SentryTracer?)!.transactionContext.name, fixture.viewControllerName)
-        XCTAssertEqual((transactionSpan as! SentryTracer?)!.transactionContext.nameSource, .component)
-        XCTAssertFalse(transactionSpan.isFinished)
+        guard let tracer = transactionSpan as? SentryTracer else {
+            XCTFail("Expected a tracer")
+            return
+        }
```

Comment:
`m`: You could use `XCTUnwrap` for all of these instead. `let tracer = try XCTUnwrap(transactionSpan)`. For more info see https://www.hackingwithswift.com/example-code/testing/how-to-check-and-unwrap-optionals-in-tests-using-xctunwrap.

Reply:
Nice, I've seen that used in the tests but forgot about it, and haven't used it myself before. I'll clean things up to use it instead!

Reply:
Yeah, that makes things much nicer. The only place I couldn't use them was within the view controller lifecycle callbacks, because they would as `throws` in swift which would require i think an inout NSError in the ObjC interface? I was just wary of changing that code.